docs/decommission.rst

```diff
@@ -0,0 +1,80 @@
+Decommission SecureDrop
+=======================
+
+The following steps will guide you through the decommissioning of your
+SecureDrop instance.
+
+#. **Put a notice in advance on your landing page to inform sources that your
+   instance will soon be retiring.**
+   You may want to direct them to other secure methods of contacting you.
+#. **Locate and create an inventory of all your hardware.**
+   - *Journalist Workstation* USBs
+   - *Admin Workstation* USB(s)
+   - *Secure Viewing Station* USB
+   - *Secure Viewing Station* computer
+   - Transfer and Export devices (USBs, optical drives, or external drives)
+   - Backup USBs/other storage media
+   - Servers
+   - Firewall
+
+   .. note:: The recommended SecureDrop setup includes only one *Secure Viewing
+      Station* USB. However, if you have been working remotely or
+      have a non-standard setup, you may have more than one *SVS USB*. It is
+      important that you locate all of these USBs, since they hold the most
+      sensitive data.
+#. **Decide whether to save a backup.**
+   If you want a backup, you can, follow our `backup guidelines`_. Once the
+   backup has been created, you can move it off of the *Admin Workstation* USB
+   and onto an encrypted device, such as a LUKS-encrypted drive. You will also
+   require the *SVS* USB and/or a backup of the SecureDrop Application Key.
+#. **Disconnect the firewall and the servers from the internet.**
+   Be sure to inform your network administrator of any changes to devices on
+   your network.
+#. **Wipe or destroy the USB drives.**
+   Because the USB drives used for SecureDrop are all LUKS-encrypted,
+   reformatting the USB drives (in particular, overwriting a portion of internal
+   storage called the **LUKS header**) should be sufficient to make any existing
+   data on those drives unrecoverable.
+
+   For example, you could use your *primary Tails USB* to launch Gnome Disks,
+   insert and identify the USB drive you are trying to erase, and reformat this
+   drive with a new, LUKS-encrypted partition, erasing the existing partition
+   data.
+
+   .. caution:: Be **very** sure you are reformatting the right drive.
+      You may want to use the Secure Viewing Station laptop for this procedure
+      to reduce the risk of accidentally erasing a drive on your regular-use
+      machine.
+
+   You may also choose to destroy the drives by physical means, such as using a
+   hammer or purpose-built shredder to pulverize or destroy the drive.
+#. **Wipe or destroy the storage drives on the servers.**
+   SecureDrop submissions are stored GPG-encrypted on the Application server.
```

Comment:
Application server->*Application Server* (standardized term)

Reply:
fixed in ff3b336